tailwind.config.js

```diff
@@ -0,0 +1,253 @@
+const { isObject, keys } = require('lodash');
```

Comment:
can we revisit how we could split the styling from the business logic so these kind of styles can be bolted on on a case by case basis?

How will the styling be changed when we use the component in a second place if it is core to the component itself?